styleguide/components/CodeBlock/CodeBlock.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import cx from 'classnames';
+import refractor from 'refractor/core.js';
+import jsxLang from 'refractor/lang/jsx';
+import rehype from 'rehype';
+
+import css from './CodeBlock.css';
+
+refractor.register(jsxLang);
```

Comment:
I think we'll need to register additional syntax's if we want to offer the ability to select them with a prop